src/sas/sasgui/guiframe/CategoryInstaller.py

```diff
@@ -170,4 +170,4 @@ def check_install(homedir = None, model_list=None):
                 CategoryInstaller._regenerate_master_dict(by_model_dict,
                                                           model_enabled_dict)
 
-            json.dump(master_category_dict, open(serialized_file, 'wb'))
+            json.dump(master_category_dict, open(serialized_file, 'w'))
```

Comment:
Out of curiosity, why was the binary flag removed?

Reply:
No, it should still be writing with binary.  I learned recently that we can use `io.open` to get the same file support in python 2 and 3.  This will allow us to open it as ascii with `'w'`, but control line endings with `newline='\n'`.